pages/podcast/index.vue

```diff
@@ -5,7 +5,7 @@ import { useRuntimeConfig } from '#app'
 const config = useRuntimeConfig()
 const apiUrl = `${config.API_URL}/api/v3/channel/shows/radiolab/recent_stories/`
 /*Algolia Search START*/
-const { result, search } = useAlgoliaSearch('radiolab') // pass your index name as param
+const { result, search } = useAlgoliaSearch(config.ALGOLIA_RADIOLAB_INDEX) // pass your index name as param
```

Comment:
nice, much better this way

Reply:
Yes, that's why I wasn't seeing the demo index.